settings.json.docker

```diff
@@ -634,5 +634,10 @@
   "customLocaleStrings": {},
 
   /* Disable Admin UI tests */
-  "enableAdminUITests": false
+  "enableAdminUITests": false,
+
+  /*
+   * Enable/Disable case-insensitive pad names.
+   */
+  "enforceLowerCasePadIds": "${ENFORCE_LOWER_CASE_PAD_IDS:false}"
```

Comment:
nit: The word "enforce" might be too strong—it implies that it is impossible to have upper-case characters.

(I wish that naming things wasn't one the hardest problems in software engineering. :slightly_smiling_face:)

Reply:
yes, lets just go with `lowerCasePadIds`

Reply:
see f91b8dbfcb7c00199fc6e94fd98a1d6a043bd505